src/main/java/net/dv8tion/jda/core/JDABuilder.java

```diff
@@ -716,6 +718,7 @@ public JDA buildAsync() throws LoginException
      *         Use {@link #build()} and {@link JDA#awaitStatus(Status)} instead
      */
     @Deprecated
+    @ReplaceWith("build().awaitReady()")
```

Comment:
Wouldn't the better ReplaceWith suggestion be `build().awaitStatus(Status)` here?